Formula/enpass-cli.rb

```diff
@@ -0,0 +1,32 @@
+class EnpassCli < Formula
+  desc "Enpass command-line client"
+  homepage "https://github.com/hazcod/enpass-cli"
+  url "https://github.com/hazcod/enpass-cli/archive/refs/tags/v1.6.0.tar.gz"
+  sha256 "25668bb30747dc566695ff7f30e65fa2e29b04f5896155a4ab03185d8f5b4111"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  resource "testdata1" do
+    url "https://raw.githubusercontent.com/hazcod/enpass-cli/250b210987aa91ef3d556b1d93b22eca7fcf5c31/test/vault.json"
+    sha256 "a01d76ed4a2f3eaf6d695bb7727f80fb9f2b44f11f5850c1387c7ac6f45eadd2"
+  end
+  resource "testdata2" do
+    url "https://raw.githubusercontent.com/hazcod/enpass-cli/250b210987aa91ef3d556b1d93b22eca7fcf5c31/test/vault.enpassdb"
+    sha256 "451f7c6d7afcd6ea505d54a99a7b7a2c8ea79186f6c6c84daae73e2a6913c1c3"
+  end
+
+  def install
```

Comment:
```suggestion
  depends_on "go" => :build

  def install
```
If my suggestions above work then we don't need these.